roles/postfix/README.md

```diff
@@ -0,0 +1,36 @@
+Role Name
+=========
+
+Creates a relay to proofpoint and creates a relay for our applications
+
+
+Role Variables
+--------------
+
+Most use for this role will involve adding your IP address to the allow list with the following variable:
+
+```bash
+relay_add_host: 1.2.3.4
+relay_remove_host: 1.2.3.4
```

Comment:
are we still using lineinfile for this? I'd prefer to see a template used, so you just update the variable with the hosts in it - any host you add to the var gets added to the config; any host you remove from the var gets removed from the config, like we do with SSH keys.